firmware/boron-ins-fsm/test/mocks/mock_ticks.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Mock implementation for time/tick functions
+ */
+
+#pragma once
+
+#include "Particle.h"
```

Comment:
Does this need all of `Particle.h` or would it also work with just `helpers.h`?

Reply:
Good point. I changed it to just include helpers.h